.github/actions/openziti-tunnel-build-action/gh-release/entrypoint.sh

```diff
@@ -44,26 +44,34 @@ fi
 for SAFE in \
     /github/workspace \
     /__w/ziti-tunnel-sdk-c/ziti-tunnel-sdk-c \
-    /mnt ; do
-        git config --global --add safe.directory ${SAFE}
+    /mnt \
+    /usr/local/vcpkg
```

Comment:
adding vcpkg checkout to safe dirs introduces the option to run the builder container as non-root, since root cloned the repo during image build